packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx

```diff
@@ -16,19 +16,19 @@ import { ReactComponent as SvgNwd } from "../../../svg/unknowndocument.svg";
 import "./BridgeIcon.scss";
 
 interface BridgeIconProps {
-  bridgeType?: IModelBridgeType;
+  bridgeType?: ConnectorTypeSynchronizationAPI;
```

Comment:
might want to replace all occurrences of the term "bridge" for "connector" in the repo

Reply:
Will do in a separate PR, I dont want to obscure this one more than it is.

Reply:
Hmm, might as well change it now, I think I already touch all the lines where this term is used...

Reply:
Updated